.azure-pipelines/azure-pipelines.yml

```diff
@@ -7,6 +7,7 @@ pr:
   autoCancel: true
   branches:
     include: [main, stable-1]
+    exclude: [release-please--*]
```

Comment:
As far as I understood the [docs](https://learn.microsoft.com/en-us/azure/devops/pipelines/repos/github?view=azure-devops&tabs=yaml#pr-triggers), `include` and `exclude` refer to the _target_ branches of the Pull Request. `release-please--*` only matches the _source_ branch of the PRs we want to exclude.

We can still merge and try if you want to.